Services/Common/OJS.Services.Common.Models/Configurations/HttpConfig.cs

```diff
@@ -0,0 +1,8 @@
+namespace OJS.Services.Common.Models.Configurations;
+
+public class HttpConfig : BaseConfig
+{
+    public override string SectionName => "Http";
+
+    public int MaxRequestSizeLimit { get; set; } = 35000000;
```

Comment:
Can you change the value to 35_000_000 and add comment for easy reading e.g. `= 35_000_000 // 35MB`

Reply:
Fixed